spec/controllers/applicants_controller_spec.rb

```diff
@@ -424,6 +424,23 @@
       expect(response.body).not_to match(/Barthelemy/)
     end
 
+    context "when there is all types of rdv_contexts statuses" do
+      before do
+        RdvContext.statuses.map do |status, _status_id|
+          create(:rdv_context, motif_category: "rsa_orientation",
+                               status: status,
+                               applicant: create(:applicant, organisations: [organisation], department: department))
+        end
+      end
+
+      it "displays all statuses in the filter list" do
+        get :index, params: index_params.merge(motif_category: "rsa_orientation")
+        RdvContext.statuses.map do |status, _status_id|
```

Comment:
nitpick: tu peux utiliser `each_key` pour itérer sur les clés du hash plutôt que `map` (qui transforme le hash en un tableau).